src/commands/generate/__tests__/serviceGeneratorTest.ts

```diff
@@ -517,7 +517,7 @@ export interface IMyService {
 
         expect(contents).not.toContain(
             `
-            /** 
+            /**
```

Comment:
tslint update caught this trailing space, which highlights that negative tests aren't great.